components/button/button-icon.js

```diff
@@ -116,7 +117,7 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 					--d2l-icon-fill-color: var(--d2l-button-icon-fill-color-hover, var(--d2l-color-tungsten));
 					background-color: var(--d2l-button-icon-background-color-hover);
 				}
-				button.focus-visible {
+				button:${unsafeCSS(getFocusPseudoClass())} {
```

Comment:
Button-icon is overriding the default `:focus-visible` styles from `buttonStyles`, which is why it needs to do this.